pkgs/development/python-modules/peewee/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, cython
+, sqlite
+}:
+
+buildPythonPackage rec {
+
+  pname = "peewee";
+  version = "3.7.0";
```

Comment:
They're at 3.7.1 now

Reply:
i cross fingers that they do not release a new version until tomorrow